CMakeLists.txt

```diff
@@ -5,9 +5,10 @@ include(CTest)
 enable_testing()
 
 add_subdirectory(src)
-add_subdirectory(test)
+add_subdirectory(tests/unit)
+add_subdirectory(tests/flow)
```

Comment:
Is there anything within tests/flow at the moment?
if not then please remove this line until we'll have flow tests to run.

Reply:
I almost finished to write the flow tests and integrate them into the repository, so I think it is ok that it is there.